data-processing/data_manipulation/llm_api_service/qa_provider_zhi_pu_ai_online.py

```diff
@@ -105,20 +105,20 @@ def generate_qa_list(
 
                             break
                         else:
-                            logger.warn('failed to get QA list, wait for 10 seconds and retry')
-                            time.sleep(10) # sleep 10 seconds
+                            logger.warn('failed to get QA list, wait for 120 seconds and retry')
+                            time.sleep(120) # sleep 120 seconds
                             invoke_count += 1
                     else:
                         logger.error(''.join([
                             f"{log_tag_const.ZHI_PU_AI} Cannot access the ZhiPuAI service.\n",
                             f"The error is: \n{response['msg']}\n"
                         ]))
-                        logger.warn('zhipuai request failed, wait for 10 seconds and retry')
-                        time.sleep(10) # sleep 10 seconds
+                        logger.warn('zhipuai request failed, wait for 120 seconds and retry')
+                        time.sleep(120) # sleep 120 seconds
                         invoke_count += 1
             except Exception as ex:
-                logger.warn('zhipuai request exception, wait for 10 seconds and retry')
-                time.sleep(10)
+                logger.warn('zhipuai request exception, wait for 120 seconds and retry')
+                time.sleep(120)
```

Comment:
better to define a constant for time interval 120

Reply:
enen,ok

Reply:
Resolved.